src/frontend/src/optimizer/plan_node/stream_hop_window.rs

```diff
@@ -37,15 +36,16 @@ impl StreamHopWindow {
         let i2o = logical.i2o_col_mapping();
         let dist = i2o.rewrite_provided_distribution(input.distribution());
 
+        let watermark_columns = i2o.rewrite_bitset(input.watermark_columns());
```

Comment:
Also, the watermark of `time_col` can also derive the watermark on `window_start` and `window_end`